test/navigationLibrary/StandardRoute/RouteSegmentCollection.spec.js

```diff
@@ -53,13 +54,21 @@ ava('.findWaypointsForSegmentName() returns an array of fixes for a given segmen
     t.true(_isEqual(result[0], ['PIRMD', null]), 'Expected to receive and array that is [`{string}`, `{string|null}`]');
 });
 
-ava('.gatherFixNamesForCollection() returns a list of names for each RouteSegmentModel in the collection', t => {
+ava('.gatherFixNamesForCollection() returns a list of entryPoint/exitPoint names for each RouteSegmentModel in the collection', t => {
     const collection = new RouteSegmentCollection(ROUTE_SEGMENTS_MOCK);
     const result = collection.gatherFixNamesForCollection();
 
     t.true(result.length === 8);
 });
 
+ava('.getAllFixNamesInUse() returns an array of fix names used by any segment in the collection', (t) => {
+    const collection = new RouteSegmentCollection(ROUTE_SEGMENTS_MOCK);
+    const result = collection.getAllFixNamesInUse().sort();
+    const expectedResult = ['BAKRR', 'BESSY', 'FIXIX', 'HITME', 'JAKER', 'JESJI', 'MDDOG', 'MINEY', 'PIRMD', 'RBELL', 'ROPPR', 'TARRK', 'WASTE'];
+
+    t.true(_isEqual(result, expectedResult));
```

Comment:
I know we do this all over the place,  but remember you can use Ava native method `t.deepEqual()` here instead of `_isEqual()`

Reply:
:+1: done